python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -112,6 +112,11 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
     if missing_cred:
         raise Exception("Failed due to missing credentials %s" % str(missing_cred))
 
+    # Make sure that all the config options are really set.
+    # Some from plugins may not have taken during startup
```

Comment:
I'm concerned by this, what config options fall into this category and how is this different to what is being done within normal Ganga bootstrap?